src/Raven.Server/Storage/Schema/Updates/Documents/From11.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Raven.Server.Documents;
+using Raven.Server.Documents.Revisions;
+using Raven.Server.Json;
+using Raven.Server.ServerWide.Context;
+using Voron.Data.Tables;
+using static Raven.Server.Documents.Revisions.RevisionsStorage;
+using static Raven.Server.Documents.DocumentsStorage;
+
+namespace Raven.Server.Storage.Schema.Updates.Documents
+{
+    public unsafe class From11 : ISchemaUpdate
+    {
+        public bool Update(UpdateStep step)
+        {
+            // When the revision are enabled and we delete a document we stored it with a 'RevisionDelete' flag. This flag was used to find the deleted revisions.
+            // Now we store the resolved conflicts as revisions, so it could be, that a deleted revision will contain flags such as 'Conflicted' or 'Resolved'.
+
+            // This change require use to change the index definition and the logic of how we find the deleted revisions.
+            // So we say that if the revision is deleted it will be stored with the 'DeletedEtag' to the etag value, 
+            // otherwise (if the revision is a document) it will be stored with 'DeletedEtag' set to 0.
+
+            step.DocumentsStorage.RevisionsStorage = new RevisionsStorage(step.DocumentsStorage.DocumentDatabase, step.WriteTx);
+
+            using (step.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+            {
+                step.WriteTx.DeleteTree("RevisionsFlagsAndEtag"); // remove the old index
+
+                foreach (var collection in step.DocumentsStorage.RevisionsStorage.GetCollections(step.ReadTx))
+                {
+                    var collectionName = new CollectionName(collection);
+                    var readTable = step.DocumentsStorage.RevisionsStorage.EnsureRevisionTableCreated(step.ReadTx, collectionName);
```

Comment:
`Ensure` will attempt to create the table, and error on the read tx
Just call `Open`, no?